ghp_import.py

```diff
@@ -74,7 +74,8 @@ def try_rebase(remote, branch):
     (rev, ignore) = p.communicate()
     if p.wait() != 0:
         return True
-    cmd = ['git', 'update-ref', 'refs/heads/%s' % branch, rev.strip()]
+    rev = rev.strip().decode('ascii')
```

Comment:
Any reason why you didn't just call `dec(rev.strip())` with the `dec` function defined earlier? That works for me. L77 should just be:

```python
cmd = ['git', 'update-ref', 'refs/heads/%s' % branch, dec(rev.strip())]
```